src/components/Blog-Footer.js

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export default function BlogFooter(props) {
```

Comment:
I would be probably keep this component in a local branch instead of merging it into master — at the moment it doesn't do anything. (Also, keeping it out of master means that if you end up not wanting to add all these features after all, you won't have to go back and remove this component)